src/screens/library/CollectionCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { Card } from 'react-native-elements';
+
+import { DiscriminatedItem } from '@graasp/sdk';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { LibraryStackNavigationProp } from '../../navigation/LibraryNavigator';
+import CollectionCreator from './CollectionCreator';
+import CollectionThumbnail from './CollectionThumbnail';
+import Tags from './Tags';
+
+type Props = {
+  item: DiscriminatedItem;
+};
+
+const CollectionCard = ({ item }: Props) => {
+  const { navigate } = useNavigation<LibraryStackNavigationProp>();
+
+  return (
+    <TouchableOpacity
+      onPress={() => {
+        navigate('CollectionStack', { itemId: item.id });
+      }}
+    >
+      {/* @ts-ignore */}
```

Comment:
🤔 I guess this is an error of using the old version of "react-native-elements" we should probably update as noticed in #106